src/app/modules/item/pages/item-edit-advanced-parameters/item-edit-advanced-parameters.component.ts

```diff
@@ -3,12 +3,8 @@ import { FormGroup, Validators } from '@angular/forms';
 import { DropdownOption } from 'src/app/modules/shared-components/components/dropdown/dropdown.component';
 import { Item } from '../../http-services/get-item-by-id.service';
 
-const MIN_MAX_ALLOW_RANGE_HOURS = 1;
-const MIN_MAX_ALLOW_RANGE_MS = MIN_MAX_ALLOW_RANGE_HOURS * 3600 * 1000;
-
-function getDefaultTimeMaxDate(timeMinDate: Date): Date {
-  return new Date(timeMinDate.getTime() + MIN_MAX_ALLOW_RANGE_MS);
-}
+const INIT_MAX_RANGE_HOURS = 1;
+const INIT_MAX_RANGE_MS = INIT_MAX_RANGE_HOURS * 3600 * 1000;
```

Comment:
use the `HOURS` constant, done for that `INIT_MAX_RANGE_HOURS* HOURS`  (actually, you can use it directly in the code, no need for a 2nd constant)

Reply:
Fixed